data_summaries/AgentInstruct.json

```diff
@@ -0,0 +1,301 @@
+{
+    "AgentInstruct-alfworld": {
+        "Unique Dataset Identifier": "AgentInstruct-alfworld",
+        "Dataset Name": "ALFworld",
+        "Paper Title": "ALFWorld: Aligning Text and Embodied Environments for Interactive Learning",
+        "Dataset URL": "https://github.com/alfworld/alfworld/tree/master/alfworld/data",
+        "GitHub URL": "https://github.com/THUDM/AgentTuning",
+        "Hugging Face URL": "https://huggingface.co/datasets/THUDM/AgentInstruct/viewer/default/alfworld",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2010.03768",
+        "Semantic Scholar Corpus ID": 222208810,
+        "Collection": "AgentInstruct",
+        "Collection URL": "https://huggingface.co/datasets/THUDM/AgentInstruct",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Instruction Following",
+            "Chain-of-Thought"
+        ],
+        "Text Sources": [
+            "crowdsourced"
+        ],
+        "Model Generated": [],
+        "Format": [
+            "Multi-turn Dialog"
+        ],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [
+            "ALFRED"
+        ],
+        "Creators": [
+            "University of Washington",
+            "Microsoft",
+            "Carnegie Mellon University"
+        ],
+        "Licenses": [
+            {
+                "License": "MIT License",
+                "License URL": "https://github.com/alfworld/alfworld"
+            }
+        ],
+        "License Notes": "The used ALFRED dataset is licensed under the MIT License https://github.com/askforalfred/alfred/tree/master and the used TextWorld is licensed under the GNU General Public License (GPL) v2.0 License. The authors only state that ALFworld is licensed under the MIT License.",
```

Comment:
Fantastic note. Thank you for adding this. Is TextWorld part of the ALFRED dataset? If it is then perhaps we should add that as one of the licenses, even if the authors didn't flag it?

Reply:
In this setting, ALFRED was extended using TextWorld. Therefore, I hve added all licenses for ALFRED, ALFworld and TextWorld. Interestingly, while checking TextWorld, I discovered by chance that the license stated in the ALFworld repository and the original TextWorld repository appeared to be different. Perhaps Microsoft updated the license later on, and the authors of ALFworld were not aware of it.